client/src/main/python/slipstream/utils/tasksrunner.py

```diff
@@ -94,12 +95,14 @@ def work(self):
 
 class ThreadWrapper(Thread):
     def __init__(self, group=None, target=None, name=None,
-                 args=(), kwargs=None, verbose=None, exc_queue=None):
+                 args=(), kwargs=None, verbose=None, exc_queue=None,
+                 daemonic=True):
         if verbose != None and int(verbose) >= 3:
             verbose = True
         super(ThreadWrapper, self).__init__(group=group, target=target,
                                             name=name, args=args,
                                             kwargs=kwargs, verbose=verbose)
+        self.setDaemon(daemonic)
```

Comment:
This is the pre-2.6 way to set a thread as a deamon.
It should be 
```python
self.daemon = daemonic
```
See https://docs.python.org/2/library/threading.html#threading.Thread.daemon